### PR TITLE
Pd-432 license prometheus alerts

### PR DESCRIPTION
**Pull request by nathan-px**

### CODE REVIEW

File: static/samples/k8s/prometheus/06-portworx-rules.yaml [hunk 1]
```diff
@@ -105,4 +105,38 @@ data:
...
+          summary: Portworx cluster member(s) is(are) down.
+      - alert: PortworxLicenseLeaseExpired
+        expr: px_alerts_license_lease_has_expired_total > 0
+        labels:
+          issue: Portworx license lease has expired.
+          severity: critical
+        annotations:
+          description: 'Portworx license lease for {{$labels.instance}} has expired.'
+          summary: 'Portworx license lease has expired on {{$labels.instance}}. Please check licenses on license server.'
+          # Do these veriables work here?
```
Comment (nathan-px) [inline static/samples/k8s/prometheus/06-portworx-rules.yaml:117]: @adityadani do these variables work here? I would think not.

File: static/samples/k8s/prometheus/06-portworx-rules.yaml [hunk 2]
```diff
@@ -105,4 +105,38 @@ data:
...
+        - alert: PortworxLicenseServerUnreachable
+          # this was missing from the paste, what is it really supposed to be?
```
Comment (nathan-px) [inline static/samples/k8s/prometheus/06-portworx-rules.yaml:119]: @adityadani The actual alert name wasn't in the ticket pasted output. What is it really?

File: static/samples/non-k8s/prometheus/px.rules [hunk 3]
```diff
@@ -108,3 +108,48 @@ Alert PortworxMemberDown
...
+  LABELS {
+    severity = "critical",
+    issue="Portworx license lease has expired",
+    }
+  ANNOTATIONS {
+    summary = "Portworx license lease has expired on {{$labels.instance}}. Please check licenses on license server.",
+    description = "Portworx license lease for {{$labels.instance}} has expired."
+  }
+
+# MISSING!!
```
Comment (nathan-px) [inline static/samples/non-k8s/prometheus/px.rules:123]: @adityadani this one missing, same as above

File: static/samples/k8s/prometheus/06-portworx-rules.yaml (hunk 1 above)
Comment (adityadani) [reply static/samples/k8s/prometheus/06-portworx-rules.yaml:117]: @shailvipx  / @prashantsr  Can you help here?

File: static/samples/k8s/prometheus/06-portworx-rules.yaml (hunk 2 above)
Comment (shailvipx) [reply static/samples/k8s/prometheus/06-portworx-rules.yaml:119]: ALERT PortworxLicenseServerUnreachable
IF px_alerts_node_could_not_contact_license_server_total > 0

File: static/samples/k8s/prometheus/06-portworx-rules.yaml (hunk 1 above)
Comment (shailvipx) [reply static/samples/k8s/prometheus/06-portworx-rules.yaml:117]: These variables will work.

**Pull request merged**